fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/source/IcebergApiSource.java

```diff
@@ -42,6 +42,9 @@ public class IcebergApiSource implements IcebergSource {
 
     public IcebergApiSource(IcebergExternalTable table, TupleDescriptor desc,
                             Map<String, ColumnRange> columnNameToRange) {
+        if (table.isView()) {
```

Comment:
`IcebergApiSource` is created in `IcebergScanNode`.
So if this is a view, I can not query it?

Reply:
Plz add comment